src/oss/confluent.sh

```diff
@@ -154,6 +154,20 @@ requirements() {
     if [[ ${status} -ne 0 ]]; then
         FORMAT_CMD="xargs -0"
     fi
+
+    # Validate Java Version
+    local java_version=$(java -version 2>&1 | sed -n ';s/.* version "\(.*\)\.\(.*\)\..*"/\1\2/p;')
+    echo "Using Java Version $java_version"
+    # Warn users if it is not Java7 or Java8, or if we can't figure it out.
+    if [[ $java_version -ne 17 && $java_version -ne 18 ]]; then
+        echo "WARNING: Java version 1.8 is recommended:"
+        echo "  See http://confluent.io for detailed version and interoperability information"
+    fi
+    # Fail fast if the version is definitely Java9
+    if [[ $java_version = 19 ]]; then
+        echo "Java 9 is unsupported at this time: Confluent CLI will exit."
```

Comment:
We need some better messaging here because AK supports Java 9.  Either allow users to run `confluent start kafka` even with java 9, or explain what the issue is (that the other components don't support it)